### DISCUSSION TITLE
A new major mode dart-ts-mode for Dart, can eglot add entry for it?

### DISCUSSION THREAD

**Discussion by dalugm** [joaotavora/eglot#1207] [Q&A] [answered]
There is new major
[mode](https://github.com/50ways2sayhard/dart-ts-mode) for editing dart
file, I tested it on my own by change the src of eglot.el from

https://github.com/joaotavora/eglot/blob/28c1c3a52e1cb7fa7260815eb53700f348d48dd5/eglot.el#L222

to this:

``` diff
-(dart-mode . ("dart" "language-server"
+((dart-mode dart-ts-mode) . ("dart" "language-server"
```

And it works fine. Can we add entry for it? thx

**Comment by joaotavora** [comment, marked as answer]
Yes, but send this as a patch to bug-gnu-emacs@gnu.org, for the lisp/progmodes/eglot.el file.
